swan-lake/development-tutorials/build-a-graalvm-executable/graalvm-executable-overview.md

```diff
@@ -36,7 +36,7 @@ The only downside is that the GraalVM native image build is a highly complicated
 
 From Ballerina 2201.7.0 (SwanLake) onwards, Ballerina supports GraalVM AOT compilation to generate standalone executables by passing the `graalvm` flag in the build command: `bal build --graalvm`. The generated executable contains the modules in the current package, their dependencies, Ballerina runtime, and statically linked native code from the JDK.
 
-Ballerina runtime, [standard libraries](/learn/ballerina-specifications/#standard-library-specifications), and the Ballerina extended modules are GraalVM-compatible. Therefore packages developed only using these libraries are also GraalVM-compatible. Furthermore, Ballerina reports warnings when the GraalVM build is executed for a project with GraalVM-incompatible packages.
+The Ballerina runtime and [Ballerina library](https://central.ballerina.io/search?q=org%3Aballerina%2Cballerinax&page=1) are GraalVM-compatible. Therefore packages developed only using these libraries are also GraalVM-compatible. Furthermore, Ballerina reports warnings when the GraalVM build is executed for a project with GraalVM-incompatible packages.
```

Comment:
isn't the `libraries` is the correct term to be used here?
```suggestion
The Ballerina runtime and [Ballerina libraries](https://central.ballerina.io/search?q=org%3Aballerina%2Cballerinax&page=1) are GraalVM-compatible. Therefore packages developed only using these libraries are also GraalVM-compatible. Furthermore, Ballerina reports warnings when the GraalVM build is executed for a project with GraalVM-incompatible packages.
```

Reply:
Saying `ballerina libraries` is incorrect as per https://github.com/ballerina-platform/ballerina-dev-website/pull/7782#issuecomment-1723704068.